### PR TITLE
[Inbox] Fixes loading indicator stuck after fetch and conversation row collapsing 

### PR DESCRIPTION
**Pull request by l2succes**
Fixes: https://github.com/artsy/collector-experience/issues/821
Fixes: https://github.com/artsy/collector-experience/issues/850
Fixes: https://github.com/artsy/collector-experience/issues/857
Fixes: https://github.com/artsy/collector-experience/issues/804

#skip_new_tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Inbox/Conversations/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "#skip_new_tests".
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  [danger-plugin-jest] Could not read test results. Danger cannot pass or fail the build.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Components/Inbox/Conversations/ConversationSnippet.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ const HorizontalLayout = styled.View`
   flex-direction: row;
 `
 
-const Card = styled(VerticalLayout)`
+const Card = styled.View`
```
Comment (l2succes) [inline src/lib/Components/Inbox/Conversations/ConversationSnippet.tsx]: The problem with the row collapsing is `VerticalLayout` includes `flex: 1` which messes with the height of the rows

Comment (mennenia) [reply src/lib/Components/Inbox/Conversations/ConversationSnippet.tsx]: 👍

**Comment by mennenia** [comment]
coooolz :) 👍

**Comment by sarahscott** [comment]
@l2succes i wonder if this PR will also solve https://github.com/artsy/collector-experience/issues/857

**Comment by l2succes** [comment]
@orta Any ideas why CI is failing here? Looks like travis is not running tests for some reason https://travis-ci.org/artsy/emission/jobs/319535516#L658

**Comment by orta** [comment]
@l2succes - it's not passing prettier

**Comment by orta** [comment]
It's likely either #905 or you've not done a `yarn install` locally after pulling, which @xtina-starr is gonna fix with https://github.com/xtina-starr/pull-lock

**Comment by sarahscott** [comment]
Another flex-related Inbox error: https://github.com/artsy/collector-experience/issues/804

**Pull request merged**